common.py

```diff
@@ -90,11 +91,19 @@ def cache_check(youtube_id):
     try:
         cache_path = ap.cache_dir + urllib.parse.quote(youtube_id) + ".json"
         ret = json.load(open(cache_path))
-        if len(ret) is CACHE_NUM:
-            # cache elements count is control value
-            return ret
-        else:
-            # cache elements count is not control value
+        if len(ret) is CACHE_ELMS:  # in case of number of cached elements is correct
+            title, time_line, time_data, total_damage, debuff_value, past_status = ret
+            if past_status // 100 == 3:
+                now = datetime.date.today()  # 現在の時刻を取得
```

Comment:
```suggestion
                now = datetime.datetime.today()  # 現在の時刻を取得
```
日付ではなく時刻の取得を想定されていると思われます。